packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -148,6 +148,33 @@ void main() {
       expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor != CupertinoColors.black), false);
     });
 
+    testWidgets('item extent default value', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoTimerPicker(
+            onTimerDurationChanged: (_) { },
+          ),
+        ),
+      );
+
+      final Iterable<CupertinoPicker> pickers = tester.allWidgets.whereType<CupertinoPicker>();
+      expect(pickers.any((CupertinoPicker picker) => picker.itemExtent != 32), false);
```

Comment:
Nit: Maybe make _kItemExtent `@visibleForTesting` and reuse it here?  Or maybe just assign 32 to a local _kItemExtent variable so it's clear where it comes from.

Reply:
Actually, maybe just remove this test?  I don't know if it's testing anything useful.  It would have passed before this PR, too.